tests/unit/development/__init__.py

```diff
@@ -0,0 +1,136 @@
+                    "- optionalNode (a list of or a singular dash component, "
+                    "string or number; optional)",
+                    "- optionalElement (dash component; optional)",
+                    "- optionalEnum (a value equal to: 'News', 'Photos'; optional)",
+                    "- optionalUnion (string | number; optional)",
+                    "- optionalArrayOf (list of numbers; optional)",
+                    "- optionalObjectOf (dict with strings as keys and values "
+                    "of type number; optional)",
+                    "- optionalObjectWithExactAndNestedDescription (optional): . "
+                    "optionalObjectWithExactAndNestedDescription has the "
+                    "following type: dict containing keys "
+                    "'color', 'fontSize', 'figure'.",
+                    "Those keys have the following types:",
+                    "  - color (string; optional)",
+                    "  - fontSize (number; optional)",
+                    "  - figure (optional): Figure is a plotly graph object. "
+                    "figure has the following type: dict containing "
+                    "keys 'data', 'layout'.",
+                    "Those keys have the following types:",
+                    "  - data (list of dicts; optional): data is a collection of traces",
+                    "  - layout (dict; optional): layout describes "
+                    "the rest of the figure",
+                    "- optionalObjectWithShapeAndNestedDescription (optional): . "
+                    "optionalObjectWithShapeAndNestedDescription has the "
+                    "following type: dict containing keys "
+                    "'color', 'fontSize', 'figure'.",
+                    "Those keys have the following types:",
+                    "  - color (string; optional)",
+                    "  - fontSize (number; optional)",
+                    "  - figure (optional): Figure is a plotly graph object. "
+                    "figure has the following type: dict containing "
+                    "keys 'data', 'layout'.",
+                    "Those keys have the following types:",
+                    "  - data (list of dicts; optional): data is a collection of traces",
+                    "  - layout (dict; optional): layout describes "
+                    "the rest of the figure",
+                    "- optionalAny (boolean | number | string | dict | "
+                    "list; optional)",
+                    "- customProp (optional)",
+                    "- customArrayProp (list; optional)",
+                    "- data-* (string; optional)",
+                    "- aria-* (string; optional)",
+                    "- in (string; optional)",
+                    "- id (string; optional)",
+                    "        ",
+                ]
+            )[i]
+        )
+
+
+def assert_flow_docstring(docstring):
+    for i, line in enumerate(docstring.split("\n")):
```

Comment:
same here, and we should move the `assert`s back into the test case whenever possible.

Reply:
TBH I found it odd as well, but I did not investigate enough to see how and when it was decided to do so. Anyway, I'll change it !